lib/tictactoe/validation.rb

```diff
@@ -0,0 +1,25 @@
+module Tictactoe
+  class Validation
+    def check_input_symbol(input)
+      if ["x", "o"].include? input
+        return input
+      else
+        puts "invalid input, symbol be either 'x' or 'o' lowercase"
+      end
+    end
+
+    def check_input_position(input)
+      if input <= 0 || input > 9
+        puts "position out of range, enter from 1 to 9"
+      else
+        return input
+      end
+    end
+
+    def check_board_position(input, board)
```

Comment:
It's not clear to me from reading the method name what the difference is between `check_input_position` and `check_board_position`. Please make the method names more descriptive.